src/module/item/base.ts

```diff
@@ -100,6 +100,10 @@ class ItemPF2e extends Item<ActorPF2e> {
             options.unshift(`${delimitedPrefix}type:${itemType}`);
         }
 
+        if (this.isOfType("consumable")) {
+            options.push(`${delimitedPrefix}type:${this.consumableType}`);
```

Comment:
Ahh very nice, I was missing knowledge of how to do this when I first did the crafting code. 🙂

Reply:
Move to consumable class and call it a subtype.